membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -49,71 +49,89 @@ class AttributeController extends Controller with LoggingWithLogstashFields {
+            case Some(attrs @ Attributes(_, Some(tier), _, _, _, _, _)) =>
+              log.info(s"$identityId is a member - $endpointDescription - $attrs")
+              onSuccessMember(attrs).withHeaders(
+                "X-Gu-Membership-Tier" -> tier,
+                "X-Gu-Membership-Is-Paid-Tier" -> attrs.isPaidTier.toString
+              )
+            case Some(attrs) =>
+              log.info(s"$identityId is a contributor - $endpointDescription - $attrs")
+              onSuccessMemberAndOrContributor(attrs)
+            case _ =>
+              onNotFound
+          }
+        case None =>
+          metrics.put(s"$endpointDescription-cookie-auth-failed", 1)
+          Future(unauthorized)
 
-  private def lookup(endpointDescription: String, onSuccessMember: Attributes => Result, onSuccessMemberAndOrContributor: Attributes => Result, onNotFound: Result) = backendAction.async { request =>
-      authenticationService.userId(request).map[Future[Result]] { id =>
-        request.touchpoint.attrService.get(id).map {
-          case Some(attrs @ Attributes(_, Some(tier), _, _, _, _, _)) =>
-            log.info(s"$id is a member - $endpointDescription - $attrs")
-            onSuccessMember(attrs).withHeaders(
-              "X-Gu-Membership-Tier" -> tier,
-              "X-Gu-Membership-Is-Paid-Tier" -> attrs.isPaidTier.toString
-            )
-          case Some(attrs) =>
-            log.info(s"$id is a contributor - $endpointDescription - $attrs")
-            onSuccessMemberAndOrContributor(attrs)
-          case _ =>
-            onNotFound
         }
-      }.getOrElse {
-        metrics.put(s"$endpointDescription-cookie-auth-failed", 1)
-        Future(unauthorized)
       }
-    }
+  }
 
 
-    private def attributesFromZuora(identityId: String, zuoraRestService: ZuoraRestService[Future], subscriptionService: SubscriptionService[Future])(implicit request: BackendRequest[AnyContent]): Future[Option[Attributes]] = {
+  private def attributesFromZuora(identityId: String, zuoraRestService: ZuoraRestService[Future], subscriptionService: SubscriptionService[Future]): Future[Option[Attributes]] = {
 
-      def withTimer[R](whichCall: String, result: Future[Disjunction[String, R]]) = {
-        import loghandling.StopWatch
-        val stopWatch = new StopWatch
+    def withTimer[R](whichCall: String, result: Future[Disjunction[String, R]]) = {
+      import loghandling.StopWatch
+      val stopWatch = new StopWatch
 
-        result.map { res: Disjunction[String, R] =>
-          val latency = stopWatch.elapsed
-          val customFields: List[LogField] = List("zuora_latency_millis" -> latency.toInt, "zuora_call" -> whichCall, "identityId" -> identityId)
-          res match {
-            case -\/(a) => logErrorWithCustomFields(s"$whichCall failed. Msg: ${a}", customFields)
-            case \/-(_) => logInfoWithCustomFields(s"$whichCall took ${latency}ms", customFields)
+      result.map { res: Disjunction[String, R] =>
```

Comment:
I think this is existing code but normally I'd avoid using two synonyms in the same code for different variables.
in this case it could be 
```scala
futureResult.map { result: Disjunction[String, R] =>
```